CHANGELOG.md

```diff
@@ -91,6 +91,8 @@ The changes are relative to the previous release, unless the baseline is specifi
   (https://crbug.com/oss-fuzz/65657).
 * ext/libjpeg.cmd now pulls libjpeg-turbo instead of libjpeg and AVIF_JPEG=LOCAL
   now expects the library dependency in ext/libjpeg-turbo/build.libavif.
+* Allow YCgCo_Re and YCgCo_Ro decoding by default. Encoding is still conditioned
```

Comment:
Thanks for writing the pull request. It would be prudent to wait until we have formally confirmed with the AOMedia Codec Working Group or the Still Image Subgroup that we can start using YCgCo-Re and YCgCo-Ro in AVIF.

Reply:
I updated https://github.com/AOMediaCodec/libavif/issues/2077 accordingly. Let's convert this PR to a draft.